packages/jupyter-ai/jupyter_ai/document_loaders/directory.py

```diff
@@ -126,16 +132,24 @@ def split(path, all_files: bool, splitter):
                 ]
                 filenames = [f for f in filenames if not f[0] == "."]
             filepaths += [Path(os.path.join(dir, filename)) for filename in filenames]
+        filepaths = [
+            fp
+            for fp in filepaths
+            if fp.suffix.lower() in {j.lower() for j in SUPPORTED_EXTS}
+        ]
+    return filepaths
 
-    for filepath in filepaths:
-        # Lower case everything to make sure file extension comparisons are not case sensitive
-        if filepath.suffix.lower() not in {j.lower() for j in SUPPORTED_EXTS}:
-            continue
 
+"""Splits files into chunks for vector db in RAG"""
```

Comment:
This is still essentially a comment and not a docstring. You can test it by mousing over in VSCode. Note how information is missing from the the pop up window. 
<img width="511" alt="image" src="https://github.com/jupyterlab/jupyter-ai/assets/26686070/5970960b-78dc-4e64-ae7b-93672633c1ad">


> "A docstring is a string literal that occurs as the first statement in a module, function, class, or method definition." 

[What is a Docstring?](https://peps.python.org/pep-0257/#what-is-a-docstring) from PEP 257. 

Please see relevant example: https://peps.python.org/pep-0257/#one-line-docstrings.

Reply:
This has been corrected and checked.